x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts

```diff
@@ -443,7 +444,10 @@ export class TelemetryReceiver implements ITelemetryReceiver {
       listId: ENDPOINT_TRUSTED_APPS_LIST_ID,
       page: 1,
       perPage: 10_000,
-      filter: undefined,
+      filter: `exception-list-agnostic.attributes.created_at > ${moment
```

Comment:
Although it's unlike to matter, this should be 
```suggestion
      filter: `exception-list-agnostic.attributes.created_at >= ${moment
```